slides/ui-customization.qmd

```diff
@@ -149,12 +149,8 @@ app_ui = ui.page_fluid(
 - We have a `navset_tab` function which organizes `ui.nav` containers into tabs
 - `ui.nav` is itself a container and allows you lay out the things on the tab
 
-## Navsets
-::: {.nonincremental}
-- It's common to put apps in tabs
-- We have a set of `navset` functions which organize `ui.nav` containers in tabs
-- `ui.nav` is itself a container and allows you lay out the things on the tab
-:::
+. . .
```

Comment:
ICYMI, you can use `. . .` to introduce a pause https://quarto.org/docs/presentations/revealjs/#incremental-lists